CI builds are broken on Windows

__Describe the Bug__

Starting yesterday, all CI builds start to break on Windows, especially our nightly: https://github.com/camunda/camunda-modeler/runs/5324718004?check_suite_focus=true

```sh
Error output:
node-pre-gyp ERR! install response status 404 Not Found on https://node-precompiled-binaries.grpc.io/grpc/v1.24.10/electron-v12.1-win32-x64-unknown.tar.gz 
node-pre-gyp WARN Pre-built binaries not installable for grpc@1.24.10 and electron@12.1.2 (electron-v12.1 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error response status 404 Not Found on https://node-precompiled-binaries.grpc.io/grpc/v1.24.10/electron-v12.1-win[32](https://github.com/camunda/camunda-modeler/runs/5324718004?check_suite_focus=true#step:5:32)-x64-unknown.tar.gz 
gyp ERR! find VS 
gyp ERR! find VS msvs_version not set from command line or npm config
gyp ERR! find VS VCINSTALLDIR not set, not running in VS Command Prompt
gyp ERR! find VS unknown version "undefined" found at "C:\Program Files\Microsoft Visual Studio\2022\Enterprise"
gyp ERR! find VS could not find a version of Visual Studio 2017 or newer to use
npm ERR! code ELIFECYCLE
npm ERR! errno 1
gyp ERR! find VS looking for Visual Studio 2015
gyp ERR! find VS - not found
gyp ERR! find VS not looking for VS2013 as it is only supported up to Node.js 8
gyp ERR! find VS 
gyp ERR! find VS **************************************************************
gyp ERR! find VS You need to install the latest version of Visual Studio
gyp ERR! find VS including the "Desktop development with C++" workload.
gyp ERR! find VS For more information consult the documentation at:
gyp ERR! find VS https://github.com/nodejs/node-gyp#on-windows
gyp ERR! find VS **************************************************************
gyp ERR! find VS 
gyp ERR! configure error 
gyp ERR! stack Error: Could not find any Visual Studio installation to use
```

__Steps to Reproduce__

<!-- Clear steps that allow us to reproduce the error. -->

1. Open [builds](https://github.com/camunda/camunda-modeler/actions)
2. See that all builds are failing on Windows while installing the dependencies, beginning yesterday ~10AM


__Expected Behavior__

Builds should run green.

__Environment__

- OS: [e.g. MacOS 10.2, Windows 10]
- Camunda Modeler Version: [e.g. 2.0.0]
- Execution Platform: [e.g. Camunda Platform, Camunda Cloud]
- Installed plug-ins: [...]

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -4,7 +4,7 @@ jobs:
   Build:
     strategy:
       matrix:
-        os: [ ubuntu-latest, macos-10.15, windows-latest ]
+        os: [ ubuntu-latest, macos-10.15, windows-2019 ]
     runs-on: ${{ matrix.os }}
 
     steps:

